zrpcd/Makefile.am

```diff
@@ -18,6 +18,9 @@ libzrpc_a_SOURCES = \
 	qzmqclient.c qzcclient.capnp.c qzcclient.c zrpc_util.c \
 	zrpc_bgp_capnp.c
 
+pkginclude_HEADERS = \
+	zrpc_os_wrapper.h zrpc_global.h
+
 noinst_HEADERS = \
```

Comment:
those files are used only for internal need.
then, I don't think this is necessary to add it in package ( implicitly that means that we provide some APIS, which is not the case).
they are not used as API to be used for other 


better to use:
noinst_HEADERS = zrpc_os_wrapper.h zrpc_global.h


also, as API, one important file is vpnservice.thrift file.
I don't know if it can be appended as pkginclude_HEADERS ?

this thrift file is the real API to use, to communicate with zrpcd.